syfertext/doc.py

```diff
@@ -354,30 +360,14 @@ def get_encrypted_token_vectors(
 
         # Encrypt the tensor using SMPC with PySyft
         token_vectors = token_vectors.fix_precision().share(
-            *workers, crypto_provider=crypto_provider, requires_grad=requires_grad
+            *workers,
+            crypto_provider=crypto_provider,
+            requires_grad=requires_grad,
+            protocol=protocol,
         )
 
         return token_vectors
 
-    def decode_logits(
```

Comment:
Why the decode logits method is removed?

Reply:
I don't remember removing it, I removed by mistake